### ISSUE TITLE
Delta printers probe error

### ISSUE THREAD

**Issue by Sab-tech-lab** [Cartographer3D/cartographer-klipper#144]
While working on getting Cartographer to work on my v400 I noticed it was missing the probe declaration.
Adding this line:
```
def get_status(self, eventtime=None):
        return {“name”: “probe”}
```
to ScannerWrapper class of the scanner.py file.

I was able to get the probe working, could you please add this small change in the next version of the software?

**Comment by krautech** [comment]
@Jomik

**Comment by Sab-tech-lab** [comment]
There's probably also a problem with the automatic z-offset calculation, but I haven't been able to figure out where the problem is yet; I suspect the problem is related to the delta kinematics and its way home that involves bringing the printer head to the maximum possible height instead of down near the print bed. I am trying to analyze what is happening in the offset calculation that seems too low.

**Comment by krautech** [comment]
I don't think carto or bed scanners are generally working on delta printers as a whole are they?

It's impossible to scan the bed correctly

**Comment by Sab-tech-lab** [comment]
> I don't think carto or bed scanners are generally working on delta printers as a whole are they?
> 
> It's impossible to scan the bed correctly

Sorry but why do you say that? My configuration works correctly, I just have problems with the automatic z-offset calculation.
![immagine](https://github.com/user-attachments/assets/0587b89b-9687-4ef3-867a-0cf977befbdc)

Bed mesh created with cartographer:
![immagine](https://github.com/user-attachments/assets/934ea941-9a16-4812-bb05-b03456d80d5b)

**Comment by Sab-tech-lab** [comment]
> Seems like two separate issues here?

Actually you are right, shall I open another issues?

**Comment by krautech** [comment]
> > Seems like two separate issues here?
> 
> Actually you are right, shall I open another issues?

Yeah this one is a PR ready for merge. Can you make another `issue` for other problem please?

**Comment by krautech** [comment]
Resolved with #156
